src/imputors/svd.jl

```diff
@@ -49,11 +46,11 @@ function impute!(data::AbstractMatrix{<:Union{T, Missing}}, imp::SVD) where T<:R
     oX = X[omask]
 
     # Fill in the original data
-    impute!(data, imp.init)
+    impute!(data, imp.init; dims=dims)
 
     C = sum(abs2, mdata - mX) / sum(abs2, mdata)
     err = mean(abs.(odata - oX))
-    @info("Before: Diff=$(sum(mdata - mX)), MAE=$err, convergence=$C, normsq=$(sum(abs2, mdata)), $(mX[1])")
+    @debug("Before: Diff=$(sum(mdata - mX)), MAE=$err, convergence=$C, normsq=$(sum(abs2, mdata)), $(mX[1])")
```

Comment:
We can take advantage of the Logging way to log arbitary values.
Shorter, and lets thigns like TensorBoardLogger work with this
```suggestion
    @debug("Before", Diff=(sum(mdata - mX)), MAE=err, convergence=C, normsq=(sum(abs2, mdata)), (mX[1])")
```

Similar below

Reply:
Cool, 👍